text/0000-project-descriptor.md

```diff
@@ -0,0 +1,150 @@
+# Meta
+[meta]: #meta
+- Name: Minimal Project Descriptor
+- Start Date: 2019-06-11
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: https://github.com/buildpack/rfcs/pull/25
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new file, `project.toml`, containing configuration for apps, services, functions, and buildpacks.
+
+# Motivation
+[motivation]: #motivation
+
+This proposal is meant to address all of the following issues:
+
+* Build metadata (i.e. information about the code being built)
+* [Application descriptor](https://github.com/buildpack/spec/issues/44)
```

Comment:
#44 includes environment variables which isn’t addressed in this proposal. These variables seemed intended to influence buildpack execution, but it could be useful to define variables to be propagated into the container environment too?